cli/spark/endpoints/cluster/cluster_create.py

```diff
@@ -128,6 +125,7 @@ def print_cluster_conf(cluster_conf):
              cluster_conf.size + cluster_conf.size_low_pri)
     log.info(">        dedicated:      %s", cluster_conf.size)
     log.info(">     low priority:      %s", cluster_conf.size_low_pri)
+    log.info("mixed mode:              %s", cluster_conf.mixed_mode)
```

Comment:
I'm not sure I would expose this. I don't think it really adds any value to the user.